app/scripts/translations.js

```diff
@@ -14,7 +14,7 @@ angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
     gettextCatalog.setStrings('nl', {});
```

Comment:
I don't know of it to cause harm. This is a temp translation anyway as it will get improved and it will change soon. I'll investigate further by then